README.md

```diff
@@ -8,3 +8,21 @@ An app to track attendance at UIUC
 * Run `npm install` in the newly-cloned directory
 * Run `npm run dev`
 * Go to http://localhost:3000 in your browser to see the application
+
+## DB Setup And Configuration
+
+### Setup Script
+
+If setting up on an install of `postgresql` that has a `postgres` user, can simple execute `utils/db_setup/psqlInit.sh`
+
+### Manual Setup
+
+If not, simply execute `utils/db_setup/dbInit.sql` using the `psql` client.
+
+For example,
+
+```
+psql <normal-postgres-args> -a -f utils/db_setup/dbInit.sql
```

Comment:
what are the normal postgres args? 😛

Reply:
This varies due to the fact that users like @JParisFerrer need to do `-h localhost` due to WSL client to windows native host. I can put an example for the most common ones.

Reply:
That'd be good - don't let people guess what those args might be!

Reply:
Added that in the last commit of this PR.